pwa-devdocs/scripts/create-compatibility-table.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * This script agenerates a PWA Studio to Magento Backend
+ * compatibility table in markdown.
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const compatibilityDefinitions = require('../../magento-compatibility.js');
+
+const outputDirectory = path.resolve('src/_includes/auto-generated');
+const outputFilename = 'magento-compatibility.md';
+const outputFile = path.resolve(outputDirectory, outputFilename);
+
+/*
+ *  Create a table row for every entry in the definition JSON.
+ */
+const tableRows = Object.keys(compatibilityDefinitions).reduce((rows, pwaVersion) => {
+    const magentoVersion = compatibilityDefinitions[pwaVersion];
+
+    const thisRow = `| ${pwaVersion} | ${magentoVersion} |\n`;
+    rows += thisRow;
+    
+    return rows;
+}, '');
+
+/*
+ *  Create the contents of the markdown file.
+ */
+const markdownContents = `| PWA Studio | Magento |
+| --- | --- |
+${ tableRows }
+`;
+
+fs.writeFileSync(outputFile, markdownContents);
```

Comment:
Very cool of you to do this. I can't wait until we generate the devdocs with Gatsby instead of Jekyll 😜